scripts/rollup/build.js

```diff
@@ -416,88 +328,77 @@ function getPlugins(
   modulesToStub,
   featureFlags
 ) {
+  const findAndRecordErrorCodes = extractErrorCodes(errorCodeOpts);
   const shims = Modules.getShims(bundleType, entry, featureFlags);
-  const plugins = [
+  const isProduction = isProductionBundleType(bundleType);
+  const isInGlobalScope = bundleType === UMD_DEV || bundleType === UMD_PROD;
```

Comment:
I’m not sure this is the best name. Maybe `isUMDBundle`?

Reply:
There may be more bundle types where top-level things are declared in global scope. In theory. :-)